python/helper-image/launcher/launcher.go

```diff
@@ -358,3 +364,11 @@ func logrusLevel(env env) logrus.Level {
 	}
 	return logrus.WarnLevel
 }
+
+func pathExists(path string) bool {
```

Comment:
Would it make sense to use os.path.exists? - https://docs.python.org/3/library/os.path.html#os.path.exists

Reply:
I wish Go had something as simple 🙄